lldb/source/Plugins/Process/Dpu/ProcessDpu.cpp

```diff
@@ -578,8 +585,10 @@ size_t ProcessDpu::UpdateThreads() { return m_threads.size(); }
 
 Status ProcessDpu::SetBreakpoint(lldb::addr_t addr, uint32_t size,
```

Comment:
Does function_group feature works with breakpoint now?
No worries if not and WIP status.
if it's not working yet, maybe return a failure if we try to breakpoint in such section would make sense for now?

same apply to unwinding.
it's a very experimental feature we understand.